common/src/main/resources/runtime.properties

```diff
@@ -37,6 +37,12 @@
 # If true, Falcon skips oozie dryrun while scheduling entities.
 *.falcon.skip.dryrun=false
 
+# Applicable only if Falcon is configured to use Oozie as it's scheduler
+# Minimum value that the Falcon Oozie Coordinator Builder will use when 
+# configuring the throttle for an Oozie coordinator. It is recommended that the
+# minimum be greater than or equal to the Oozie default of 12.
+*.falcon.oozie.coordinator.throttle.min=12
```

Comment:
Sure! Sorry for the delay.

Question, though. Is it OK that this pull request is going to have more than one commit? I see [in the How to Contribute](https://cwiki.apache.org/confluence/display/FALCON/How+To+Contribute) that it needs to be a single commit.